.github/workflows/build-container.yml

```diff
@@ -0,0 +1,61 @@
+name: Release ingress-netperf container image
+
+on:
+  pull_request:
+    paths:
+      - 'containers/**' # Trigger only when Dockerfile changes in a pull request
+
+env:
+  CONTAINER_REGISTRY: ${{ github.repository_owner == 'cloud-bulldozer' && 'quay.io' }}
+  GO_VER: 1.19
+
+jobs:
+
+  build-and-push-container-images:
+    name: Build and Push Container Images
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+    
+    steps:
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup docker-buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to the ${{ env.CONTAINER_REGISTRY }} Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.CONTAINER_REGISTRY }}
+          username: ${{ env.CONTAINER_REGISTRY == 'quay.io' && secrets.QUAY_USER || github.actor }}
+          password: ${{ env.CONTAINER_REGISTRY == 'quay.io' && secrets.QUAY_TOKEN || secrets.GITHUB_TOKEN }
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.CONTAINER_REGISTRY }}/${{ github.repository_owner }}/netperf
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
```

Comment:
Are these steps actually required now?

Reply:
Login to the ${{ env.CONTAINER_REGISTRY }} Container Registry will be required.
Not the meta step

Reply:
This is updated and the metadata step is not required.